Makefile

```diff
@@ -330,27 +326,6 @@ FLEET_DESKTOP_VERSION ?= unknown
 desktop-windows:
 	GOOS=windows GOARCH=amd64 go build -ldflags "-H=windowsgui -X=main.version=$(FLEET_DESKTOP_VERSION)" -o fleet-desktop.exe ./orbit/cmd/desktop
 
-# Build desktop executable for Linux.
-#
-# Usage:
-# FLEET_DESKTOP_VERSION=0.0.1 make desktop-linux
-#
-# Output: desktop.tar.gz
-desktop-linux:
-	docker build -f Dockerfile-desktop-linux -t desktop-linux-builder .
-	docker run --rm -v $(shell pwd):/output desktop-linux-builder /bin/bash -c "\
-		mkdir /output/fleet-desktop && \
-		go build -o /output/fleet-desktop/fleet-desktop -ldflags "-X=main.version=$(FLEET_DESKTOP_VERSION)" /usr/src/fleet/orbit/cmd/desktop && \
-		cp /usr/lib/x86_64-linux-gnu/libayatana-appindicator3.so.1 \
-		/usr/lib/x86_64-linux-gnu/libayatana-ido3-0.4.so.0 \
-		/usr/lib/x86_64-linux-gnu/libayatana-indicator3.so.7 \
-		/lib/x86_64-linux-gnu/libm.so.6 \
-		/usr/lib/x86_64-linux-gnu/libdbusmenu-gtk3.so.4 \
-		/usr/lib/x86_64-linux-gnu/libdbusmenu-glib.so.4 \
-		/output/fleet-desktop && cd /output && \
-		tar czf desktop.tar.gz fleet-desktop && \
-		rm -r fleet-desktop"
```

Comment:
This one is needed actually :)

The one I proposed removing is the one I previously proposed `desktop-linux-build`.

Reply:
Oh gotcha ok!